explorer/graphql/app/schema.graphql

```diff
@@ -122,9 +122,9 @@ type XReceipt {
 
 type Query {
   xblock(sourceChainID: BigInt!, height: BigInt!): XBlock
-  xblockrange(amount: BigInt!, offset: BigInt!): [XBlock]!
+  xblockrange(from: BigInt!, to: BigInt!): [XBlock]!
```

Comment:
I'm not sure if `to` should be required. Maybe you can fetch a block since a know block height.

Reply:
I'm going to hold off on this until we get pagination, if we don't have `to` be required right now, we would return from -> genesis